common/src/KokkosFFT_transpose.hpp

```diff
@@ -113,15 +113,15 @@ void _prep_transpose_view(InViewType& in, OutViewType& out,
 
 template <typename ExecutionSpace, typename InViewType, typename OutViewType,
           std::enable_if_t<InViewType::rank() == 1, std::nullptr_t> = nullptr>
-void _transpose(const ExecutionSpace& exec_space, InViewType& in,
-                OutViewType& out, [[maybe_unused]] axis_type<1> _map) {}
+void _transpose(const ExecutionSpace&, InViewType&, OutViewType&,
+                axis_type<1>) {
+  // FIXME: Comment why empty?
```

Comment:
That is true. It may be better to remove this and avoid calling `_transpose` function in `transpose` for 1D case?